src/Client.php

```diff
@@ -265,6 +296,16 @@ public function getLastMessage()
     }
 
     /**
+     * Was the last write successful
+     *
+     * @return bool
+     */
+    public function wasSuccessful()
+    {
+        return $this->written;
+    }
```

Comment:
Not currently... Probably should inject a `WriterInterface` and use that for testing.